src/meta/src/hummock/compaction/mod.rs

```diff
@@ -162,11 +167,34 @@ impl CompactStatus {
         Some(compact_task)
     }
 
+    pub fn is_trivial_move_task(task: &CompactTask) -> bool {
```

Comment:
need some doc for `trivial_move` for understanding of logic. (When reading the code for the first time, it is not easy to understand each branch)

Reply:
I will add it in next PR